projects/ecc-diff-fuzzer/Dockerfile

```diff
@@ -16,10 +16,12 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER p.antoine@catenacyber.fr
-RUN apt-get update && apt-get install -y make cmake
+RUN apt-get update && apt-get install -y make cmake curl bzip2 autoconf automake libtool sed
```

Comment:
Do you really need to install curl here? E.g. here it's not installed: https://github.com/google/oss-fuzz/blob/6c0458a14cea99e7de83fb0cde60eeb6635ce56e/projects/gnupg/Dockerfile

Reply:
ok with `ADD`